.github/workflows/pr_swift_lint.yaml

```diff
@@ -0,0 +1,84 @@
+name: pr_swift_lint
```

Comment:
For here, I'm expecting that there's a pre-prepared workflow that can be subbed in for parts of this, but is looking really good all the same :)